### ISSUE TITLE
TypeError: create_sample_data_csv() missing 1 required positional argument: 'file_name' in ml_service/pipelines/diabetes_regression_build_train_pipeline.py", line 62

### ISSUE THREAD

**Issue by martin-weber** [microsoft/MLOpsPython#253]
In "Publish Azure Machine Learning Pipeline" of "Model CI Pipeline" I got the TypeError: create_sample_data_csv() missing 1 required positional argument: 'file_name' in ml_service/pipelines/diabetes_regression_build_train_pipeline.py", line 62

I was able to fix it by exchanging the lines 62 and 65 in ml_service/pipelines/diabetes_regression_build_train_pipeline.py and add the file_name as parameter to create_

Before:		
```python
    # Check to see if dataset exists
    if (dataset_name not in aml_workspace.datasets):
        # This call creates an example CSV from sklearn sample data. If you
        # have already bootstrapped your project, you can comment this line
        # out and use your own CSV.
        create_sample_data_csv()

        # Use a CSV to read in the data set.
        file_name = 'diabetes.csv'
```
After fixing:
```python
    # Check to see if dataset exists
    if (dataset_name not in aml_workspace.datasets):
        # Use a CSV to read in the data set.
        file_name = 'diabetes.csv'

        # This call creates an example CSV from sklearn sample data. If you
        # have already bootstrapped your project, you can comment this line
        # out and use your own CSV.
        create_sample_data_csv(file_name)
```

**Comment by martin-weber** [comment]
PR #252 would also close this issue by removing the file_name argument

**Comment by eedorenko** [comment]
Fixed by #252
